src/mappings/Reputation/datasource.yaml

```diff
@@ -10,3 +10,5 @@ eventHandlers:
     handler: handleMint
   - event: Burn(indexed address,uint256)
     handler: handleBurn
+  - event: OwnershipTransferred(indexed address,indexed address)
```

Comment:
why is that needed ?

Reply:
I thought it made sense since DAOToken and Avatar track this event and instantate the entity in the store based off of it. Also I htink there could be cases where a new DAO is deployed without any initial REP holders, and we would still want the ReputationContract entity to exist in the store for completeness sake.

A good example of this is when you have a token holder DAO that relies on token holders redeeming reputation. When the DAO is first deployed there may be zero REP holders.